Description: Use libc++ "extensive" hardening mode
Author: Daniel Richard G. <skunk@iSKUNK.ORG>

Make build/config/compiler/BUILD.gn add

  -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_EXTENSIVE

to the build.

--- a/build/config/c++/c++.gni
+++ b/build/config/c++/c++.gni
@@ -75,7 +75,7 @@ declare_args() {
 # The saigo libc++ is distinct from the custom hermetic libc++. However, since
 # the Chrome team controls the saigo toolchain, it is safe to unconditionally
 # enable libc++ hardening there as well.
-use_safe_libcxx = use_custom_libcxx && enable_safe_libcxx
+use_safe_libcxx = enable_safe_libcxx
 
 # libc++abi needs to be exported from executables to be picked up by shared
 # libraries on certain instrumented builds.
